week1/MyCard/app/src/main/java/com/example/mycard/MainActivity.kt

```diff
@@ -0,0 +1,186 @@
+            verticalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier
+                .fillMaxSize()
+                .background(Color(0xFF103650)),) {
+
+            Column(modifier = Modifier
+                .fillMaxHeight()
+                .weight(3f, false),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Image(
+                    modifier = Modifier.size(200.dp),
+                    contentScale = ContentScale.Crop,
+                    painter = imagePainter,
+                    contentDescription = stringResource(id = R.string.desc_android)
+                )
+                Text(
+                    text = name,
+                    color = Color.White,
+                    fontSize = 50.sp,
+                    modifier = Modifier.padding(
+                        bottom = 16.dp,
+                    )
+                )
+                Text(
+                    text = job,
+                    color = Color(0xFF3ddc84),
+                    fontWeight = FontWeight.Bold,
+                    textAlign = TextAlign.Justify,
+                    modifier = Modifier.padding(
+                        start = 16.dp,
+                        end = 16.dp
+                    )
+                )
+            }
+
+            Column(modifier = Modifier
+                .fillMaxWidth()
+                .weight(1f, false),
+                verticalArrangement = Arrangement.Bottom) {
+
+                Divider(
+                    color = Color.White,
+                    modifier = Modifier
+                        .fillMaxWidth()  //fill the max height
+                        .width(1.dp)
+                )
+
+                Row(modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 10.dp, bottom = 10.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Center) {
+                    Icon(
+                        Icons.Rounded.Phone,
+                        tint = Color(0xFF3ddc84),
+                        contentDescription = stringResource(id = R.string.desc_phone),
+                        modifier = Modifier.weight(1.5f))
+                    Text(
+                        text = phonenumber,
+                        color = Color.White,
+                        fontSize = 20.sp,
+                        modifier = Modifier.weight(3.5f)
+                    )
+                }
+
+                Divider(
+                    color = Color.White,
+                    modifier = Modifier
+                        .fillMaxWidth()  //fill the max height
+                        .width(1.dp)
+                )
+
+                Row(modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 10.dp, bottom = 10.dp),
```

Comment:
```suggestion
                    .padding(vertical = 10.dp),
```

`vertical`을 통해 `top`과 `bottom`을 동시에 지정할 수 있습니다!